package.json

```diff
@@ -49,11 +49,12 @@
     "date-fns": "2.10.0",
     "ejs": "3.0.1",
     "express": "4.17.1",
-    "highlight.js": "^10.4.1",
+    "highlight.js": "^9.18.3",
```

Comment:
do this one needs to be rebased with master so we don't downgrade the version?

Reply:
That specific commit(75ddcfd) can be removed actually, if highlight.js error was fixed on master

Reply:
rebased to master